src/moodle-sentence.ts

```
@@ -0,0 +1,109 @@
+ *
+ * @slot - This element has a slot
+ * @csspart button - The button
+ */
+@customElement('moodle-sentence')
+export class MoodleSentence extends LitElement {
+  @query('.container')
+  container!: HTMLDivElement
+
+  #words: string[] = [];
+
+  #correctVersion: string = '';
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)